demisto_sdk/commands/common/hook_validations/playbook.py

```diff
@@ -346,6 +348,29 @@ def check_script_id(self, script_id_used_in_task, id_set_scripts):
         """
         return any([script_id_used_in_task in id_set_dict for id_set_dict in id_set_scripts])
 
+    def check_integration_command(self, integration_id_used_in_task, id_set_integrations,
+                                  empty_integration_id=True):
+        """
+        Checks if integration id and command exists in at least one of id_set's dicts
+        Args:
+            integration_id_used_in_task (str):  integration id from playbook
+            id_set_integrations (list): all integrations of id_set
+            empty_integration_id (bool): Whether integration_id is legal as None.
+             i.e.: |||Command is legal or not. true - legal, false - not
```

Comment:
the example us correct, but better explain that it excepts the case that the command does not include the brand/integration name.